pkg/api/persistentvolume/util.go

```diff
@@ -50,6 +50,10 @@ func VisitPVSecretNames(pv *api.PersistentVolume, visitor func(string) bool) boo
 		if source.ISCSI.SecretRef != nil && !visitor(source.ISCSI.SecretRef.Name) {
 			return false
 		}
+	case source.StorageOS != nil:
+		if source.StorageOS.SecretRef != nil && !visitor(source.StorageOS.SecretRef.Name) && !visitor(source.StorageOS.SecretRef.Namespace) {
```

Comment:
Do I need both ways: visitor(name) and if namespace set, visitor(namespace, name)?